packages/near-membrane-dom/src/browser-realm.ts

```diff
@@ -79,7 +81,13 @@ function createIframeVirtualEnvironment(
         keepAlive = false,
         liveTargetCallback,
         // eslint-disable-next-line prefer-object-spread
-    } = ObjectAssign({ __proto__: null }, options);
+    } = options;
+
+    const trustedCreateScript =
+        typeof options.trustedCreateScript === 'function'
+            ? options.trustedCreateScript
+            : (v: string) => v;
```

Comment:
☝️ You can add the `identity` function to the near-membrane-shared Function.ts (this is replaced on the lws side of things with it's own "shared" package so aligns with the other PR.

Reply:
The reason why I am reluctant to tie this to a utility is because I don't want to impact this functionality by any change in the utility or any validation it does or will do. I believe it is better to have it explicitly declared as a function here, which is actually the fallback for not providing the option to the VE, than using the `identity` function. It may not read as pretty as a 1 liner ternary expression but it is independent of the utility. I believe this independence weights more...

Reply:
I might follow up with a PR to reuse stuff as the syncing between `@locker/near-membrane-shared` can be less fun and distracting. If this is more about signatures we might be able to reuse the trusted types feature test in our `shared` library.